GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/DehydrateTests.cs

```diff
@@ -176,279 +176,264 @@ public void DehydrateShouldFailOnWrongDiskLayoutVersion()
 
-            this.PlaceholdersShouldNotContain(folderToDehydrate, Path.Combine(folderToDehydrate, "Program.cs"));
-            GVFSHelpers.ModifiedPathsShouldNotContain(this.Enlistment, this.fileSystem, Path.Combine(folderToDehydrate, "App.config").Replace(Path.DirectorySeparatorChar, TestConstants.GitPathSeparator));
+            this.PlaceholdersShouldNotContain(folderToDehydrate, fileToReadThenDehydrate);
+            GVFSHelpers.ModifiedPathsShouldNotContain(this.Enlistment, this.fileSystem, fileToWriteThenDehydrate.Replace(Path.DirectorySeparatorChar, TestConstants.GitPathSeparator));
 
-            this.PlaceholdersShouldContain(folderNotDehydrated, Path.Combine(folderNotDehydrated, "GVFSLock.cs"));
-            GVFSHelpers.ModifiedPathsShouldContain(this.Enlistment, this.fileSystem, Path.Combine(folderNotDehydrated, "Enlistment.cs").Replace(Path.DirectorySeparatorChar, TestConstants.GitPathSeparator));
+            this.PlaceholdersShouldContain(folderToNotDehydrate, fileToReadThenNotDehydrate);
+            GVFSHelpers.ModifiedPathsShouldContain(this.Enlistment, this.fileSystem, fileToWriteThenNotDehydrate.Replace(Path.DirectorySeparatorChar, TestConstants.GitPathSeparator));
 
             this.Enlistment.UnmountGVFS();
 
-            fileToRead.ShouldNotExistOnDisk(this.fileSystem);
-            fileToWrite.ShouldNotExistOnDisk(this.fileSystem);
-            fileNotDehydrated.ShouldBeAFile(this.fileSystem);
-            fileNotDehydrated2.ShouldBeAFile(this.fileSystem);
+            fileToReadThenDehydrateBackingPath.ShouldNotExistOnDisk(this.fileSystem);
+            fileToWriteThenDehydrateBackingPath.ShouldNotExistOnDisk(this.fileSystem);
+            fileToReadThenNotDehydrateBackingPath.ShouldBeAFile(this.fileSystem);
+            fileToWriteThenNotDehydrateBackingPath.ShouldBeAFile(this.fileSystem);
         }
 
-        [TestCase]
+        [TestCase]
         public void FolderDehydrateNestedFoldersChildBeforeParent()
         {
-            string folderToDehydrate1 = Path.Combine("GVFS", "GVFS.Mount");
-            string folderToDehydrate2 = "GVFS";
-            string fileToRead1 = this.Enlistment.GetVirtualPathTo(Path.Combine(folderToDehydrate1, "Program.cs"));
-            string fileToRead2 = this.Enlistment.GetVirtualPathTo(Path.Combine(folderToDehydrate2, "GVFS.UnitTests", "Program.cs"));
-            this.fileSystem.ReadAllText(fileToRead1);
-            this.fileSystem.ReadAllText(fileToRead2);
+            string childFolderToDehydrate = Path.Combine("GVFS", "GVFS.Mount");
+            string parentFolderToDehydrate = "GVFS";
+            string fileToReadInChildFolder = Path.Combine(childFolderToDehydrate, "Program.cs");
+            string fileToReadInParentFolder = Path.Combine(parentFolderToDehydrate, "GVFS.UnitTests", "Program.cs");

```

Comment:
Naming nit: This file path is not in the parent folder (`GVFS`), it's in a second child folder (`GVFS.UnitTests`)

Reply:
Changed to `InOtherChildFolder` -- good catch!  Thanks!